react-app/ccf/src/components/applications/AdminCoverPageModal.tsx

```
@@ -0,0 +1,72 @@
+
+const AdminCoverPageModal = ({application, isOpen, onClose}: CoverPageModalProps) => {
+
+  const [pdfLink, setPdfLink] = useState<any>();
+  const [reportLink, setReportLink] = useState<any>();
+  const [reportMsg, setReportMsg] = useState<string>("");
+
+    useEffect(() => {
+      if (isOpen) {
+        getCurrentCycle().then(currentAppCycle => {
+            downloadPDFsByName([application.file]).then((links) => {
+                if (links && links[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)